Client/Frontend/Browser/Toolbars/UrlBar/TabLocationView.swift

```diff
@@ -74,11 +74,17 @@ class TabLocationView: UIView {
     }
     
     private func updateLockImageView() {
-        let wasHidden = lockImageView.isHidden
-        lockImageView.isHidden = !contentIsSecure
-
-        if wasHidden != lockImageView.isHidden {
-            UIAccessibility.post(notification: .layoutChanged, argument: nil)
+        lockImageView.isHidden = false
+        
+        switch secureContentState {
+        case .localHost:
+            lockImageView.isHidden = true
+        case .secure:
+            lockImageView.tintColor = #colorLiteral(red: 0.3764705882, green: 0.3843137255, blue: 0.4, alpha: 1)
```

Comment:
Nit: This could be moved into the default section above the `switch` statement, next to `isHidden = false` and it wouldn't have to be duplicated across `.unknown` and `.secure`.

Reply:
Even better to support strict typing, `case .secure, .unknown`, just clump them together. That way we'll continue to get compiler checks.

Reply:
Fixed.